Bust Config.rb

```diff
@@ -156,7 +164,31 @@
     face_name: "Yarra emo2",
     face_indexes: [3],
   },
+  # Topless young Simon needs a special beardless bust
+  {
+    character_name: "Simon topless young",
+    face_name: "face002b topless",
+    face_indexes: [7],
+  },
+  {
+    character_name: "Simon topless young",
+    face_name: "face002b topless dark eyes",
+    face_indexes: [7],
+  },
 ]
+
+# map from clothed Simon to corresponding topless Simon
+Busty::TOPLESS_SIMON_MAP = {
+  "1 Simon dark eyes" => "1 Simon topless dark eyes",
+  "1 Simon dark eyes2" => "1 Simon topless dark eyes2",
+  "1 Simon dark" => "1 Simon topless",
+  "1 Simon dark2" => "1 Simon topless2",
+  "face002b" => "face002b topless",
+  "face002b2" => "face002b topless2",
+  "face002b dark" => "face002b topless dark eyes",
+  "face002b dark2" => "face002b topless dark eyes2",
+}
+
```

Comment:
While I've given up on finding a more concise way to configure that feature, a loop will at least allow to keep most config in one spot:
```rb
Busty::TOPLESS_SIMON_MAP.each do |key, value|
  Busty::FACE_TO_BUST[value] = "Simon topless"
end
```

Reply:
Done.